src/main/resources/feedbackQuestionContribSubmissionFormTemplate.html

```diff
@@ -11,11 +11,12 @@
         <div class="modal-content">
             <div class="modal-header">
                 <button type="button" class="close" data-dismiss="modal">&times;</button>
-                <h4 class="modal-title">More info about the 'Equal Share' scale</h4>
+                <h4 class="modal-title">More info about the <code>Equal Share</code> scale</h4>
             </div>
             <div class="modal-body">
-                <p>Equal share is a relative measure of individual contribution to a team.</p>
-                <p>For example, in a 3 person team, 'equal share' means a third of the work done.</p>
+                <p><code>Equal share</code> is a relative measure of individual contribution to a team task.</p>
+                <p>For example, in a 3-person team, <code>equal share</code> means a third of the work done.</p>
```

Comment:
Suggestion to capitalise this (i.e. `Equal share`) to match the option shown in the dropdown.